### ISSUE TITLE
Default command can't be chained with pluck

### ISSUE THREAD

**Issue by jonasbeyer** [rethinkdb/rethinkdb-ts#67]
**Describe the bug**

Chaining the `default` command with `pluck`  causes a TypeScript error which says that there is no compatible signature.

> TS2349: This expression is not callable. Each member of the union type '(<U>(value: U) => RDatum<Partial<unknown>[] | U>) | (<U>(value: U) => RDatum<Partial<any> | U>)' has signatures, but none of those signatures are compatible with each other.

**To Reproduce**

```
const user = await r
  .table("users")
  .get("")
  .pluck("id", "role")
  .default(null)
  .run();
```

**Expected behavior**

It should be possible to chain both commands to provide a fallback value. 
Otherwise, if a document doesn't exist, RethinkDB will throw an exception.

See: https://github.com/rethinkdb/rethinkdb/issues/3381

**Comment by atassis** [comment]
@jonasbeyer i'll check and try to fix this in next 2 days

**Comment by atassis** [comment]
@jonasbeyer deployed a fix in 2.4.5. Please, check and ping me

**Comment by atassis** [comment]
Reopen if it's not fixed
